python/dlisio/ext/core.cpp

```diff
@@ -1083,11 +1083,20 @@ PYBIND11_MODULE(core, m) {
         return recs;
     });
 
-    m.def( "parse_objects", []( const std::vector< dl::record >& recs ) {
+    m.def( "parse_objects", []( const std::vector< dl::record >& recs,
+                                dl::error_handler& errorhandler ) {
         std::vector< dl::object_set > objects;
         for (const auto& rec : recs) {
             if (rec.isencrypted()) continue;
-            objects.push_back( dl::object_set( rec ) );
+
+            try {
+                objects.push_back( dl::object_set( rec ) );
+            } catch (const std::exception& e) {
+                const auto context = "load (object set creation)";
```

Comment:
I find all these `context = load`a bit weird. Yes they are generally used from load, but doesn't have to be. Maybe do, `context = "{function} {what the function essentially does}"`? Like here: `context = "core.parse_objects: Construct object sets"`. 

This also applies to findoffsets, findfdata, readfdata, extract and object_set.parse.

Sidenote, this functions doesn't really parse objects does it? It just initialises object set